examples/scale-to-zero-with-keda/assets/scaling-scenario.drawio

```diff
@@ -0,0 +1,266 @@
+<mxfile host="app.diagrams.net" modified="2024-01-12T09:38:08.784Z" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36" etag="0TI7ytqyLkQEG066GDOh" version="22.1.17" type="device">
```

Comment:
Is this still using transparent background? Our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md#background) want white background, with light blue / light green secondary backgrounds for the environments in which something happens.